environment-dev.yml

```diff
@@ -19,12 +20,12 @@ dependencies:
   - matplotlib
   - netCDF4
   - numpy
-  - pandas!=1.5.3
+  - pandas !=1.5.3
   - pygeos
   - pyyaml
   - rechunker
   - shapely
-  - xarray
+  - xarray <2023.3.0  # See https://github.com/Ouranosinc/xscen/pull/173
```

Comment:
I disagree with pinning a package like this, _especially_ xarray!

The bug is in xclim, and comes from a change in a semi-private function in xarray (issue 7645 there).

Reply:
Understood. How do we proceed? `xarray !=2023.3.0`?

Reply:
I imagine if we can push changes to `xclim`, we can simply pin `xclim` to the next version? Will changes in `xclim` be backwards-compatible with `xarray`?

Reply:
Done, I made an Issue.

Reply:
Awesome. I'll add the changes to the changelog and update the issue to consult, then merge. Thanks!